superset/charts/api.py

```diff
@@ -39,9 +41,12 @@
     ChartPostSchema,
     ChartPutSchema,
     get_delete_ids_schema,
+    thumbnail_query_schema,
 )
 from superset.constants import RouteMethod
 from superset.models.slice import Slice
+from superset.tasks.thumbnails import cache_chart_thumbnail
+from superset.utils.selenium import ChartScreenshot
```

Comment:
I wonder if the ChartScreenshot class should be part of a package that isn't named after the underlying implementation? I can see a future where we want to remove Selenium from the stack - other languages have far better tools for this system, and I hope some of them will be ported soon. It would be nice to be able to replace the underlying implementation without touching files from other modules.

Reply:
Totally agree